wlc/__init__.py

```diff
@@ -512,6 +518,15 @@ def list(self):
         self.ensure_loaded("translations_url")
         return self.weblate.list_translations(self._attribs["translations_url"])
 
+    def add_translation(self, language):
+        """Creates a new translation in the component"""
+        self.ensure_loaded("translations_url")
+        language_list = [l["language"]["code"] for l in self.list()]
+        if language in language_list:
+            raise ContentExists()
```

Comment:
It would be nice to specify what content exactly exists - so basically raise with a more informative message such as "language X already exists in component Y".

Reply:
Since @superDross was unable to replicate this issue I think it's probably ok for us to just remove this check (and the associated exception) entirely. No sense cluttering up the codebase, and it avoids creating an additional race condition.